spice/tests/rv_gui.py

```diff
@@ -137,7 +135,7 @@ def run(vt_test, test_params, env):
         act.x_active(vmi_c)
         act.x_active(vmi_g)
         ssn = test.open_ssn(test.name_c)
```

Comment:
Could you please make a new PR to open session with:

ssn = act.new_ssn(test.vmi_c) ??